src/core/orderbook.js

```diff
@@ -123,6 +125,13 @@ function OrderBook(remote,
     self.updateFundedAmounts(transaction);
   }
 
+  function ledgerClosed(message) {
+    if (message.ledger_index === self._ledgerLastNotified) {
+      // We had transaction notifications, so notify and recompute
```

Comment:
We might do something similar with all the `offer_added` events, renaming them to `_offer_added` and buffering these events until ledger close.

@shekenahglory  

How are the offer_added etc events used in ripple-charts (if at all)?